leaflet-maplibre-gl.js

```diff
@@ -193,10 +209,18 @@
                 this._glMap._actualCanvas = this._glMap._canvas;
             }
 
+
             // treat child <canvas> element like L.ImageOverlay
             var canvas = this._glMap._actualCanvas;
             L.DomUtil.addClass(canvas, 'leaflet-image-layer');
             L.DomUtil.addClass(canvas, 'leaflet-zoom-animated');
+
+            // hide the map until it is loaded, to prevent flickering
```

Comment:
Just to make sure I understand the result, this means the map will be visible after the first render.

When does the first render occur? Immediately after adding to leaflet or is it only after the first interaction?

Reply:
Returning to this I'm not seeing the issue anymore so I've removed this code.